### ISSUE TITLE
Use other parcellation atlas instead of DK atlas

### ISSUE THREAD

**Issue by ShuyangNie** [alegiac95/Imaging-transcriptomics#30]
Dear alegiac95:

Thank you for developing imaging-transcriptomics! But I hope to use other parcellation atlas rather than DK atlas since they provide more detailed regional information. I noticed you have added possibility to use other parcellation, but I don't know what to do. Can you give me any advice? Thank you again!

Best regards!
Shuyang Nie

**Comment by iamjoostjanssen** [comment]
I agree that this would constitute a major increase in funcionality. Users with a custom atlas could provide the custom_regionXexpression matrix (from Abagen) and the (geodesic) centroids of the regions (in fsaverage space).
